Makefile

```diff
@@ -40,6 +41,7 @@ swagger-clientstub:
 
 test:
 	${SWAGGER_CMD} validate ./api/swagger-spec/swagger.yml
+	${DEV_ENV_CMD} shellcheck $(SHELL_SCRIPTS)
```

Comment:
Could you put the `shellcheck` line under a separate "test-style" target that is required by the "test" target? That's our general convention, like [in the router Makefile](https://github.com/deis/slugrunner/blob/master/Makefile#L63) for example.